app/views/shared/_header.html.erb

```diff
@@ -1,20 +1,16 @@
-<nav class="navbar navbar-dark bg-primary">
+<nav class="l-header wrap-full">
   <button class="navbar-toggler hidden-sm-up" type="button" data-toggle="collapse" data-target="#exCollapsingNavbar2">
     &#9776;
   </button>
-  <div class="collapse navbar-toggleable-xs" id="exCollapsingNavbar2">
```

Comment:
It's already implemented with the mobile design, have a look when you can. @tiagojsag